examples/example_landing_zone_s3_and_codepipeline/main.tf

```diff
@@ -1,7 +1,7 @@
 module "example_landing_zone_s3_and_codepipeline" {
   source    = "../../modules/landing_zone"
-  version   = "0.0.8"
-  root_path = "${path.module}"
+  root_path = path.module
+  output_path = pathexpand("~/.terrahub/output.json")
```

Comment:
please remove `output_path` as input parameter. update the internal scripts to use `~/.terrahub/cache/output/[project_code]/output.json` that can be used across modules ;)